tests/epyccel/test_epyccel_sets.py

```diff
@@ -77,3 +77,32 @@ def clear_complex():
     python_result = clear_complex()
     assert python_result == pyccel_result
 
+def test_copy_int(language):
+    def copy_int():
+        se = {1,2,4,5}
+        se.copy()
+        return se
+    epyccel_copy = epyccel(copy_int, language = language)
+    pyccel_result = epyccel_copy()
+    python_result = copy_int()
+    assert python_result == pyccel_result
+
+def test_copy_float(language):
+    def copy_float():
+        se = {5.7, 6.2, 4.3, 9.8}
+        se.copy()
+        return se
+    epyccel_copy = epyccel(copy_float, language = language)
+    pyccel_result = epyccel_copy()
+    python_result = copy_float()
+    assert python_result == pyccel_result
+
+def test_copy_complex(language):
+    def copy_complex():
+        se = {7j, 6j, 9j}
+        se.copy()
+        return se
+    epyccel_copy = epyccel(copy_complex, language = language)
+    pyccel_result = epyccel_copy()
+    python_result = copy_complex()
+    assert python_result == pyccel_result
```

Comment:
I think it would be worth adding checks in this file to ensure that the elements of the returned set have the correct type

Reply:
Fixed : aa7fadc

Reply:
Fixed : 74c27b9

Reply:
You seem to have missed integers

Reply:
Fixed : 8743b3f